src/main/scala/io/iohk/ethereum/blockchain/sync/BlacklistSupport.scala

```diff
@@ -12,17 +12,17 @@ trait BlacklistSupport {
 
   def scheduler: Scheduler
 
-  private val maxSize = 1000
```

Comment:
Maybe we could get this value from config?

Reply:
problem is that this trait is shared across many different components, so the question arises what config. Also some of those componets should probably use differnt values.